tests/functional/status.test.js

```diff
@@ -13,9 +13,10 @@ const {find} = require('lodash');
 
     const modules = [];
 
-    (harnessConfigs.testRemoteSwarm ? remoteSwarmHook() : localSwarmHooks({configOptions: {max_swarm_storage: testParams.maxSwarmStorage}}));
+    (harnessConfigs.testRemoteSwarm ? remoteSwarmHook() : localSwarmHooks({createDB: false, configOptions: {max_swarm_storage: testParams.maxSwarmStorage}}));
 
     before('make status request', async function () {
+        await this.api._createDB(Math.floor(testParams.maxSwarmStorage/testParams.numberOfNamespaces));
```

Comment:
What's this for?

Reply:
I've revised this to just create a DB of `testParams.namespaceSize` and added a guard to throw an error if the swarm config `max_swarm_storage` is insufficient for the tests